include/mgp.hpp

```diff
@@ -2590,6 +2653,26 @@ inline bool Node::operator==(const Node &other) const { return util::NodesEqual(
 
 inline bool Node::operator!=(const Node &other) const { return !(*this == other); }
 
+inline const std::string Node::ToString() const {
+  std::string labels{""};
+  for (auto label : Labels()) {
+    labels += ":" + std::string(label) + " ";
+  }
+  std::string properties = "";
+  std::map<std::string, Value> properties_map{Properties()};
+  const size_t map_size = properties_map.size();
+  size_t i = 0;
```

Comment:
`decltype(map_size)` will give you type here

Reply:
It will give me a const type, so no bueno